src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -42,6 +42,14 @@ const ACCEPTED_DOCS_PATHNAMES = [
 	'tools',
 	'vagrant-cloud',
 	'intro',
+	'cdktf',
+	'cli',
+	'cloud-docs',
+	'enterprise',
+	'internals',
+	'language',
+	'plugin',
+	'registry',
```

Comment:
@zchsh am I missing anything?

Reply:
Between the existing entries here and the ones you've added, this definitely covers all of Terraform's `rootDocsPaths` 👀  ✅